code/game/gamemodes/cult/runes.dm

```diff
@@ -208,13 +209,24 @@
 						log_admin("[M]([ckey(M.key)]) ghosted/disconnected less than a minute after having been converted to the cult! ([T.x],[T.y],[T.z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[T.x];Y=[T.y];Z=[T.z]'>JMP</a>)")
 			return 1
 		else
-			if(jobban_isbanned(M, "cultist"))
-				M.Sleeping(300)//putting them to sleep for 5 minutes.
+			if(is_convertable_to_cult(M.mind) && jobban_isbanned(M, "cultist")) //Not implanted, but cultbanned
+				var/turf/T = get_turf(M)
+				T.turf_animation('icons/effects/effects.dmi',"rune_teleport")
+				M.unequip_everything() //Piñata
+				//death(M) //toggles SPS from going off or not.
+				sleep(1) //Ensure everything has time to drop without getting deleted
+				qdel(M)
```

Comment:
Straight qdel is way too cheesy and unsatisfying. This is sacrificing somebody to another dimension to something that wants to consume their mind, body, and soul.

Gibbing would leave a pile of blood, which would be thematic as it's a blood-based cult, whereas ashing leaves behind unremovable piles of bones (Something which isn't used that much, besides supermatter ashing, and the main narsie summoning) so there is some consequence and some need for tact when deciding where to sacrifice somebody.